### PR TITLE
Add N&N for Windows Defender auto fix and remove general note about it

### PR DESCRIPTION
**Pull request by HannesWell**
Part of https://github.com/eclipse-platform/eclipse.platform.ui/pull/1453

### CODE REVIEW

File: news/4.31/platform.html [hunk 1]
```diff
@@ -321,8 +321,27 @@ <h2>General Updates </h2>
...
+    <!-- https://github.com/eclipse-platform/eclipse.platform.ui/pull/1453 -->
+    <td class="title">Windows Defender auto configuration</td>
+    <td class="content">
+      <p>
+        On Mirosoft Windows 10 or later the Microsoft Defender can significantly slow-down the startup and overall performance of Eclipse-based applications.
+        The Eclipse IDE can now detect if Microsoft Defender is active, inform you about that and if wanted can exclude itself from future Defender scans.
+      </p>
+      <p><img src="images/windows-defender-autofix_startup-check.png" alt="Windows Defender autofix startup check"/></p>
+      <p>
+        On Windows the <code>General &#62; Startup and Shutdown</code> preference page offers further information
```
Comment (BeckerWdf) [inline news/4.31/platform.html:338]: We don't address preference pages consistently.
Here we use <code></code> on entries above we use <em>Appearance</em> (even without a full path to that page.

Which way should be choose?

Comment (HannesWell) [reply news/4.31/platform.html]: I have no all unified all preference references I found to use `->`.
In the end I think there is not much difference between `->` and `>`, it should just be used uniformly.

**Review by BeckerWdf — APPROVED** [review]

File: news/4.31/platform.html [hunk 2]
```diff
@@ -321,8 +321,32 @@ <h2>General Updates </h2>
...
+      <p><img src="images/windows-defender-autofix_startup-check.png" alt="Windows Defender autofix startup check"/></p>
+      <p>
+        On Windows the <code>General -&#62; Startup and Shutdown</code> preference page offers further information 
+        and allows to disable the startup check for new installations and to run the exclusion check for the running installation again.
+      </p>
+      <p><img src="images/windows-defender-autofix_preference-page.png" alt="Windows Defender autofix preference page"/></p>
+      <p>
+        Creators of custom Eclipse-based products that use the <code>org.eclipse.ui.ide.workbench</code> application and want to suppress the new startup check for their product
+        can disable it by adding the following entry to their product's <em>preferenceCustomization</em> properties file:<br>
+        <code>org.eclipse.ui/windows.defender.check.skip=true</code>
```
Comment (HannesWell) [inline news/4.31/platform.html:345]: This is about to be changed to a more expressive name with https://github.com/eclipse-platform/eclipse.platform.ui/pull/1693
```suggestion
        <code>org.eclipse.ui/windows.defender.startup.check.skip=true</code>
```

Comment (HannesWell) [reply news/4.31/platform.html]: Done.

**Comment by HannesWell** [comment]
Thanks @BeckerWdf for your review.

**Pull request merged**